time stamp
---
robot_position_loader/BBox3D[] robot_bbox_vec
